modules/dnn/test/test_int8_layers.cpp

```diff
@@ -2,6 +2,7 @@
 // It is subject to the license terms in the LICENSE file found in the top-level directory
 // of this distribution and at http://opencv.org/license.html.
 
+/*
 #include "test_precomp.hpp"
```

Comment:
The file contains formatting changes only. Let's revert it to reduce branch conflicts.

Reply:
Tests in this file are **ALL** using on-fly quantization. It is not realistic to convert all test models to quantized ones. I think we will remove these tests eventually and create new ones which test existing int8-quantized layers with handcrafted model and data.

Reply:
No, no need to touch this file, just revert changes here.

Reply:
Cannot use multi-line comment `/**/` to comment out the whole file if those not formatted to use `//` instead.

Reply:
I got your point on comments. The major issue I see is gap in quantized layers testing and technical dept.
I propose the following:
- Drop large networks tests, e.g.  yolo for now.
- convert existing test cases with single layer, a.k.a. `testLayer` to quanitzed and push them to extra as separate folder.

Reply:
> convert existing test cases with single layer

There are still too many test layers. There are 42 `Test_Int8_layers` instances and each can have >10 test cases, resulting in >100 test cases in total. I propose to test existing int8 layers instead, e.g. one, two or more test cases for each int8 layers, depending on parameters.

Reply:
> Cannot use multi-line comment /**/ to comment out the whole file

because need to use `#if 0` instead.

Reply:
> > Cannot use multi-line comment /**/ to comment out the whole file
> 
> because need to use `#if 0` instead.

Thank you! It is much better than commenting indeed. I used `#if 0` instead of commenting in the latest comment.